### PR TITLE
Docs: Refine README.md

### PR DESCRIPTION
**Pull request by riandyrn**
I think not everyone can easily realized the benefit of writing maintainable code.

So in this PR I attempt to position this repo as getting started guidelines to write production-grade code for Haraj projects.

Let me your opinion about this, @harunalfat.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,31 +4,35 @@
...
 
 The secret technique actually lies in the software architecture we choose for these projects: [Hexagonal Architecture](./docs/reference/hex-architecture.md).
 
-In this repo we will share to you our knowledge on this special architecture through simple server-client game named `Hex Monscape`. We are using [Hexagonal Architecture](./docs/reference/hex-architecture.md) to structure the game server code while implementing it using [Go](https://go.dev/) & for the web client we implement it using [Vue 3](https://vuejs.org/).
+Since this repo is intended as getting started guidelines for anyone who wants to know how we write production code for Haraj projects, we will try our best to explain the technique in simple & fun ways. 
+
+In order to do that, we will be using simple server-client game named `Hex Monscape`. We are using [Hexagonal Architecture](./docs/reference/hex-architecture.md) to structure the game server code while implementing it using [Go](https://go.dev/). For the web client we implement it using [Vue 3](https://vuejs.org/).
```
Comment (harunalfat) [inline README.md:13]: The added description is giving more clarity to people what we intended to share with this repo. But I think, we also need more emphasis onto how people will be more interest to read this complete project as a whole.

Game is one point that catches many people interest. I think we should label this repo more of a "Simple Game for Haraj Engineering Onboarding", so people will have something in mind like "Oooh, they learn to write a game when onboarding? Interesting, I need to check this out".

Giving more catchy introduction like a screenshot of the game itself even better I think, to make people stay and read the repo. These are my two cents @riandyrn  😄

**Review by harunalfat — APPROVED** [review]
The wording is spot on I think, LGTM 💯 

We might also need to update the "About" description,
image

**Review by isdzulqor — APPROVED** [review]
LGTM. Awesome 🎉

File: README.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+[![Go Reference](https://pkg.go.dev/badge/github.com/Haraj-backend/hex-monscape.svg)](https://pkg.go.dev/github.com/Haraj-backend/hex-monscape)
+[![Go Report Card](https://goreportcard.com/badge/github.com/Haraj-backend/hex-monscape)](https://goreportcard.com/report/github.com/Haraj-backend/hex-monscape)
+[![coverage](https://img.shields.io/badge/coverage-82.2%25-brightgreen)](./.output/cov-report/coverage.out)
 
-Welcome to this repo!
+`Hex-Monscape` is a simple turn-based game created for Haraj Solutions Team onboarding process.
 
-In this repo you will learn about Haraj Solutions Team's secret technique in writing maintainable code for Haraj projects. 🥷🏻💥
+Inside the game source code, you can find the team's secret technique in writing production-grade code for Haraj web services that help serve millions of users daily in Saudi Arabia, despite only having a handful of team members. 🥷🏻💥
```
Comment (knightazura) [inline README.md:9]: Minor suggestion here, how if we put Haraj URL (haraj.com.sa) in `Haraj` word in this sentence?

"...team's secret technique in writing production-grade code for **[Haraj](https://haraj.com.sa)** web services..."

**Pull request merged**